sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -117,25 +117,42 @@ should be used.
 To run SYCL  applications on OpenCL devices, OpenCL implementation(s) must be
 present in the system.
 
-To execute SYCL tests on Intel devices following runtimes should be installed:
+The following `GPU` runtimes should be installed to run SYCL applications on  Intel `GPU` devices:
```

Comment:
The links below are not to exact version of the runtime, these are the links to the web-pages where Linux/Windows versions of GPU runtime can be downloaded.
I think we have to update the wording to make it clear.
NOTE: recommended version should separately provided in addition to these links.

Reply:
Ok, I re-phrased to the followinig:

> The `GPU` runtime that is needed to run SYCL application on Intel `GPU` devices
> can be downloaded from the following web pages:

I also do not understand what you mean by this: "recommended version should separately provided in addition to these links.".
How I can recommend without knowing what user has. For example, here: https://github.com/intel/compute-runtime/releases all recommendations are on that page and there is no need to recommend something here on this Gettin Started Page.

Reply:
Current version of the [document](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedWithSYCLCompiler.md#install-low-level-runtime) has quite important information, removed by this patch:
```
Please, refer to the [the Release Notes](../ReleaseNotes.md) for recommended Intel runtime versions.
```
As of today there are 78 release of NEO runtime available at https://github.com/intel/compute-runtime/releases.
Unfortunately, not all of these versions are compatible with the latest version of SYCL implementation.

We should guide our users and specify "working" versions of GPU runtime.

Reply:
Ok, I will restore the reference to Release Notes.

Regarding the recommendations and what version are working..., even I don't know, which are working and which are not. So, I don't seem a right person for writing that specifics here.

Reply:
Basically idea is that we should have instructions here without specific details on the timestamps. Or we should say clearly that timestamps given are just an examples.
Every month we are publishing release notes which has recommended timestamp per target. You can also mention this here.

Reply:
In my opinion restoring this statement is more than enough:
"Please, refer to [the Release Notes](../ReleaseNotes.md) for recommended Intel runtime versions."

Reply:
Probably yes.